src/views/TechnicalAssistance/Homepage.tsx

```diff
@@ -91,9 +93,12 @@ function Homepage() {
       },
       {
         Header: t<string>('table.header.submissionDate'),
-        accessor: 'createdAt',
+        accessor: 'form.submittedAt', // This is what breaks the Column type arg
```

Comment:
You can pass a function to `accessor` and return the nested value.  You wont need the ts ignore

```
accessor: ( { form } ) => form.submittedAt,
```

Reply:
I understand your idea, but this part has a trick. The `accessor` prop is used for many table features, sorting being the one that matters here. This patch will work around react-table's typescripting, but will break the existing working sorting behavior, which will also break some unit tests.

It's usually best overall to use accessor in a simple way to lessen effects.

I'd say let's leave this for now since the feature is working and the typescript that should be working can run into issues likes these, where generics on plugin code can sometimes be unwantingly restrictive.

For this pr I do not want to spend any more time and follow the chain of javascript fixes on things that shouldn't be this roundabout, in this case the next step would be to fix this initial sortBy here https://github.com/CMSgov/easi-app/blob/5b02c5d8b63d1b514966cebbc92b8b50f7d0e2be/src/views/TechnicalAssistance/Homepage.tsx#L131
or looking into column sort properties further to handle a complex accessor.